sparkbox/apprentices-page-next
# feat: Add Apply page

## Description

This PR adds a new page, found on `/apply`.

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. Visit the PR site, and navigate to `/apply`.
